main.tex

```diff
@@ -1065,8 +1073,8 @@ \subsubsection*{Provenance of individual \wnt{} simulation models}
 % Explicit calibration and validation of simulation models
 When looking at the CSM{} and VSM{} activities, we detect that only a few simulation models were both calibrated and validated based on wet-lab data, namely those of \lee{}~\cite{Lee2003}, Kogan et al. (2012)~\cite{Kogan2012}, and Haack et al. (2015)~\cite{Haack2015}.
 Some authors just calibrated their simulation models (\cite{Rodriguez2007, Mazemondet2012, Wang2013, Chen2014, Padala2017, Staehlke2020}) and in two studies (\cite{Cho2006, Haack2020}) they were only validated (having assumed plausible ranges for parameter values).
-The simulations models developed in \cite{Kruger2004, Kim2007, vanLeeuwen2007, Wawra2007, Goldbeter2008, vanLeeuwen2009, Mirams2010} have neither been calibrated nor validated explicitly, because they have used parameter values from other studies.
-Sick et al. (2006)~\cite{Sick2006} have used arbitrary parameter values in their simulation models.
+The simulations models developed in \cite{Kruger2004, Kim2007, vanLeeuwen2007, Wawra2007, Goldbeter2008, vanLeeuwen2009, Mirams2010} had neither been calibrated nor validated explicitly, because they have used parameter values from other studies.
```

Comment:
tense

Reply:
maybe use past and perfect? or past perfect and past? or past perfect and perfect, as you did?

Reply:
"The simulations models developed in \cite{Kruger2004, Kim2007, vanLeeuwen2007, Wawra2007, Goldbeter2008, vanLeeuwen2009, Mirams2010} had neither been calibrated nor validated explicitly, because they used parameter values from other studies."?

Reply:
oh yeah removing the "have" from "have used" is a good idea, I missed that while reading this paragraph